egs/peoples_speech/ASR/prepare.sh

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+nj=32
+stage=-1
+stop_stage=100
+
+# Split data/set to a number of pieces
+# This is to avoid OOM during feature extraction.
+num_per_split=4000
+
+# We assume dl_dir (download dir) contains the following
+# directories and files. If not, they will be downloaded
+# by this script automatically.
+#
+#  - $dl_dir/peoples_speech
+#      This directory contains the following files downloaded from
+#       https://huggingface.co/datasets/MLCommons/peoples_speech
+# 
+#     - test
+#     - train
+#     - validation
+#
+#  - $dl_dir/musan
+#      This directory contains the following directories downloaded from
+#       http://www.openslr.org/17/
+#
+#     - music
+#     - noise
+#     - speech
+
+dl_dir=$PWD/download
+
+. shared/parse_options.sh || exit 1
+
+# vocab size for sentence piece models.
+# It will generate data/lang_bpe_xxx,
+# data/lang_bpe_yyy if the array contains xxx, yyy
+vocab_sizes=(
+  # 5000
+  # 2000
+  # 1000
+  500
+)
+
+# All files generated by this script are saved in "data".
+# You can safely remove "data" and rerun this script to regenerate it.
+mkdir -p data
+
+log() {
+  # This function is from espnet
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%d %H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+log "dl_dir: $dl_dir"
+
+if [ $stage -le 0 ] && [ $stop_stage -ge 0 ]; then
+  log "Stage 0: Download data"
+
+  # If you have pre-downloaded it to /path/to/peoples_speech,
+  # you can create a symlink
+  #
+  #   ln -sfv /path/to/peoples_speech $dl_dir/peoples_speech
+  #
+  if [ ! -d $dl_dir/peoples_speech/train ]; then
+    git lfs install
+    git clone https://huggingface.co/datasets/MLCommons/peoples_speech
+  fi
+
+  # If you have pre-downloaded it to /path/to/musan,
+  # you can create a symlink
+  #
+  #   ln -sfv /path/to/musan $dl_dir/
+  #
+  if [ ! -d $dl_dir/musan ]; then
+    lhotse download musan $dl_dir
+  fi
+fi
+
+if [ $stage -le 1 ] && [ $stop_stage -ge 1 ]; then
+  log "Stage 1: Prepare People's Speech manifest"
+  # We assume that you have downloaded the People's Speech corpus
+  # to $dl_dir/peoples_speech
+  mkdir -p data/manifests
+  if [ ! -e data/manifests/.peoples_speech.done ]; then
+    lhotse prepare peoples-speech -j $nj $dl_dir/peoples_speech data/manifests
+    touch data/manifests/.peoples_speech.done
+  fi
+fi
+
+if [ $stage -le 2 ] && [ $stop_stage -ge 2 ]; then
+  log "Stage 2: Prepare musan manifest"
+  # We assume that you have downloaded the musan corpus
+  # to data/musan
+  mkdir -p data/manifests
+  if [ ! -e data/manifests/.musan.done ]; then
+    lhotse prepare musan $dl_dir/musan data/manifests
+    touch data/manifests/.musan.done
+  fi
+fi
+
+if [ $stage -le 3 ] && [ $stop_stage -ge 3 ]; then
+  log "Stage 3: Preprocess People's Speech manifest"
```

Comment:
Please change the comment for this stage. It is the same as stage 1 at present.

Reply:
Stage 1 is prepare, and Stage 3 is preprocess